test/e2e/dra/test-driver/README.md

```diff
@@ -44,6 +44,50 @@ kubelet<->dynamic resource allocation plugin interaction.
 
 `app` is the driver itself with a very simple implementation of the interfaces.
 
+## Deployment
+
+### `local-up-cluster.sh`
+
+To try out the feature, build Kubernetes, then in one console run:
+```console
+FEATURE_GATES=DynamicResourceAllocation=true KUBELET_RESOLV_CONF="/etc/resolv-9999.conf" DNS_ADDON="coredns" CGROUP_DRIVER=systemd CONTAINER_RUNTIME_ENDPOINT=unix:///var/run/crio/crio.sock LOG_LEVEL=6 ENABLE_CSI_SNAPSHOTTER=false API_SECURE_PORT=6444 ALLOW_PRIVILEGED=1 PATH=/nvme/gopath/src/k8s.io/kubernetes/third_party/etcd:$PATH ./hack/local-up-cluster.sh -O
```

Comment:
It would probably be better to set KUBELET_RESOLV_CONF to "/var/run/systemd/resolve/resolv.conf" instead of "/etc/resolv-9999.conf".

ENABLE_CSI_SNAPSHOTTER is false by default in /hack/local-cluster-up.sh, so no need to set it

Reply:
I'm unsure how complete I should make the instructions for local-up-cluster.sh here. I can leave out everything that is not needed for DRA (KUBELET_RESOLV_CONF, DNS_ADDON, CGROUP_DRIVER, CONTAINER_RUNTIME_ENDPOINT, LOG_LEVEL, ENABLE_CSI_SNAPSHOTTER, API_SECURE_PORT) but then will others still be able to bring up the cluster?

There is no good documentation for all the settings that may have to be tweaked, at least none that I am aware of.

Reply:
The ./hack/local-up-cluster.sh has default values, I've noticed that this PR was merged after I submitted the comments, sorry. I'm preparing new PR to fix this doc and comments below.